railties/Rakefile

```diff
@@ -32,6 +32,7 @@ namespace :test do
 
     dirs = (ENV["TEST_DIR"] || ENV["TEST_DIRS"] || "**").split(",")
     test_files = dirs.map { |dir| "test/#{dir}/*_test.rb" }
+    testopts = ENV["TESTOPTS"] ? ENV["TESTOPTS"].split(/[\s]+/) : []
```

Comment:
`ENV["TESTOPTS"].to_s.split`? I'd probably move the variable assignment (and rename it to test_options) to above `dirs` with a separating newline. These 3 lines seem to be working on the same thing.

Reply:
Thank you. I've just pushed everything you suggest. Is there an advantage to using `to_s` in addition to avoiding the ternary?